default.json

```diff
@@ -32,4 +33,4 @@
       ]
     }
   ]
-}
+}
```

Comment:
I want to preserve final new lines for consistency.

Reply:
Added final line in the file!
[Add final line in json files](https://github.com/hatena/renovate-config/pull/81/commits/fe4814defd7901a522e88fe3ec1ab4bc856eccfa)